dbms/src/Common/UTF8Helpers.cpp

```diff
@@ -75,15 +75,10 @@ static int wcwidth(wchar_t wc)
     switch (width)
     {
         case widechar_nonprint:
-            [[fallthrough]];
         case widechar_combining:
-            [[fallthrough]];
         case widechar_unassigned:
-            return 0;
```

Comment:
What is the logic here? Default `width` is 0?

Reply:
It's a bug, that I introduced and it was found by our test!